verifier/src/qe_identity.rs

```diff
@@ -248,7 +249,7 @@ impl SignedQeIdentity {
     ///     let time = DateTime::from_system_time(SystemTime::now()).unwrap();
     ///     ```
     ///   or equivalent
-    pub fn verify(self, key: Option<&VerifyingKey>, time: DateTime) -> Result<(), Error> {
+    pub fn verify(self, key: Option<&VerifyingKey>, time: Option<DateTime>) -> Result<(), Error> {
```

Comment:
IFF `impl Into<Option<DateTime>>` is used elsewhere I would suggest doing it here too because it's public, even though it's likely to only be used by evidence which knows what it has
```suggestion
    pub fn verify(self, key: Option<&VerifyingKey>, time: impl Into<Option<DateTime>>) -> Result<(), Error> {
```